### ISSUE TITLE
[Bug] Background color of `button` for `pressed` state in `brand` hierarchy does not change in dark mode

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#904]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 12 Pro

### OS Version

iOS 26.0 (beta 4)

### App version

0.18.0

### App build number

28

### OUDS iOS library version embeded in the app

0.18.0 (develop)

### Tokens libraries versions embeded in the app

Core OUDS 1.3.0, core Orange 1.1.0, core Sosh 1.1.0, brand Orange 1.3.0, brand Sosh 1.3.0, brand Orange Business Tools 1.3.0, brand Orange Inverse 1.3.0

### App build type

debug

### App orientation

Portrait

### Device language

French

### App language

French

### Describe the issue

The _background color_ for _pressed_ state of _button_ component in _brand_ hierarchy does not have the suitable value in some cases: it is not possible to see the _pressed_ state.

Failing cases are:
- Orange theme in dark color scheme
- Orange Busines Tools theme in dark color scheme
- Orange Inverse theme in light and dark color scheme
- Sosh theme in light color scheme

See video attached

https://github.com/user-attachments/assets/bb488fa6-f75a-46d6-ac56-0cf8b1cf9847

### Expected Behavior

Have a _pressed_ state color different than the _enabled_ color

Maybe a fix must be done in tokens libraries

### Accessibility settings

None
